docs/configuration.md

```diff
@@ -1207,6 +1207,25 @@ Apart from these, the following properties are also available, and may be useful
 <tr>
   <td><code>spark.ui.view.acls</code></td>
   <td>Empty</td>
+</tr>
+<tr>
+  <td>spark.executor.multiPerWorker</td>
```

Comment:
This is fundamentally a worker property, not an executor property, as it's the worker that reads it and makes decisions based on it, right?  If that's the case, this shouldn't be under the `spark.executor` namespace.